tests/test_show_users.py

```diff
@@ -0,0 +1,20 @@
+import snowflake.connector.cursor
```

Comment:
I don't mind splitting this out but can I suggest:
1. Updating the README.md which currently only points at test_fakes.py
1. Having a coarser grain. In the limit, if we end up with a file per command, that'll be a lot of files and potentially hard to navigate. _test_show.py_ containing all the SHOW functionality I think would be fine.

Reply:
fwiw In the existing limit, I found the majority of tests in test_fakes and test_transforms to be hard to navigate. My personal preference would be to only colocate things that are likely to fail together...but it's not my library.

I was thinking perhaps `test_users.py`, which would end up colocating tests for `SHOW USERS` which will inevitably also end up inside any tests for `CREATE USER`